statick_tool/rsc/spotbugs-security.xml

```diff
@@ -1,5 +1,5 @@
 <FindBugsFilter>
```

Comment:
Should this (and the closing tag) be `SpotBugsFilter`?

Reply:
Surprisingly, no! Upstream kept it as `FindBugsFilter` for compatibility, and as far as I can tell they don't support `SpotBugsFilter` as a tag name.